samples/Kconfig

```diff
@@ -25,6 +25,13 @@ config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
 	  Add the empty_app_core as a child image for the given sample.
 	  Used by samples that only require SOC_NRF5340_CPUNET.
 
+config NCS_SAMPLE_EMPTY_APP_CORE_PATH
```

Comment:
This option does not make sense IMO. There is one empty app core sample, and it will always be in the same directory. Instead we should create a new option which allows the user to point to an arbitrary sample to be included as a child image for a configurable domain.

That way out-of-tree users which are not running as a module are free to set the prj.conf to include child images for other domains (either net or app).

I can try to come up with a proposal.